spec/02-integration/06-invalidations/02-core_entities_invalidations_spec.lua

```diff
@@ -987,13 +987,20 @@ for _, strategy in helpers.each_strategy() do
         assert.res_status(200, res_1)
         assert.equal("2", res_1.headers["Dummy-Plugin"])
 
-        assert_proxy_2_wait({
```

Comment:
Could you please explain why does `assert_proxy_2_wait` not work for this flaky test?